hl7/4.2/modules/ROOT/pages/index.adoc

```diff
@@ -733,42 +172,59 @@ For example:
 
 The generated ACK messages have MSH data set up for sending back to the sender of the original message, so you don't need to change anything in the data to perform the send.
 
-If you include an ACK message schema in your configuration, that schema is used to both receive ACK messages and generate ACK messages. If you don't specify an ACK schema, the standard `hl7/v2_5_1/ACK.esl` schema is used by default.
+If you include an ACK message schema in your configuration, that schema is used to both receive and generate ACK messages. If you don't specify an ACK schema, the standard `hl7/v2_5_1/ACK.esl` schema default is used.
 
 === Reading and Validating HL7 ER7 Messages
```

Comment:
This really doesn't belong in the index. It might be deserving of its own topic. You can assess that later when you are doing more of the in depth edit.

Reply:
Moved to Examples.